src/CodiceFiscale/Calculator.php

```diff
@@ -119,7 +121,7 @@ private function calculateSmallString($consonants, $string)
     /**
      * Calculate the birth date and the gender.
      *
-     * @returns Returns the birth date and gender part of the codice fiscale.
+     * @returns string Returns the birth date and gender part of the codice fiscale.
```

Comment:
You could remove **Returns** here and use an uppercase **T** for the initial **the**.